internal/weldr/api.go

```diff
@@ -877,7 +878,8 @@ func (api *API) blueprintsNewHandler(writer http.ResponseWriter, request *http.R
 		return
 	}
 
-	api.store.PushBlueprint(blueprint)
+	commitMsg := "Recipe " + blueprint.Name + ", version " + blueprint.Version + " saved."
+	api.store.PushBlueprint(blueprint, commitMsg)
```

Comment:
This hunk belongs with the previous commit, but that's not a big deal. Just keep it in mind for next time.